src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -354,6 +354,12 @@ $externalMargin: 8px;
 		// header is filled with primary or image background
 		filter: none !important;
 		color: var(--color-primary-text) !important;
+
+		// NOTE when migrating to Vue3, while not deprecated, a better solution would be to style the child slotted element
+		// LINK https://stackoverflow.com/a/55368933
+		:deep * {
+			cursor: pointer !important;
+		}
```

Comment:
This is already covered by `NcButton`. And in header menu icons it works where apps use `NcButton` as a trigger without `NcButton`. Would it be possible to also use `NcButton` instead of `<button>` in this component?

Reply:
> NOTE when migrating to Vue3, while not deprecated, a better solution would be to style the child slotted element

BTW, `:slotted` would not be enough here. It works only on root elements in slot, like with `> *`.

Reply:
Hello, and happy 'beginning of the work week'! But I agree, 

> Would it be possible to also use <NcButton> instead of <button> in this component?

I think that using our own native button would be better. Will use in the new PR!